tests/example.at

```diff
@@ -1,5 +1,5 @@
 # Souffle - A Datalog Compiler
-# Copyright (c) 2017, The Souffle Developers. All rights reserved.
+# Copyright (c) 2018, The Souffle Developers. All rights reserved.
```

Comment:
Don't update the original copyright year

Reply:
Fixed in feb692275c6358a67396810e843d583af4e2f7ec